src/resources/PrivilegeEvaluator/PrivilegeEvaluatorInterfaces.ts

```diff
@@ -0,0 +1,31 @@
+import {PrivilegeModel} from '../BaseInterfaces';
+
+export interface PrivilegeEvaluatorModel {
+    /**
+     * Whether the request has been approved.
+     */
+    approved?: boolean;
+    /**
+     * A privilege request evaluation report.
+     */
+    evaluationReport?: PrivilegeRequestEvaluationReportModel;
+    /**
+     * The unique identifier of the organization in which the target privilege applies.
+     */
+    organizationId?: string;
+    /**
+     * The unique identifier of the organization in which the target privilege applies.
+     */
+    requestedPrivilege: PrivilegeModel;
+}
+
+export enum PrivilegeRequestEvaluationReportModel {
+    operationForbiddenForAnonymousAuthentication = 'OPERATION_FORBIDDEN_FOR_ANONYMOUS_AUTHENTICATION',
```

Comment:
Where did you get those values ? Is it from documentation ?

Reply:
Yes, it's from the doc
https://docs.coveo.com/en/6/cloud-v2-api-reference/authorization-server-api#operation/evaluateUsingPOST_2
![image](https://user-images.githubusercontent.com/12199712/129580111-763b9e98-220c-4a34-ad21-3de4c235e8e4.png)